world_universities_and_domains.json

```diff
@@ -130977,5 +130980,149 @@
       "uet.edu.al"
     ],
     "country": "Albania"
+  },
+  {
+    "web_pages": [
+      "https://rkgit.edu.in/"
+    ],
+    "name": "Raj Kumar Goel Insititue Of Technology",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "rkgit.edu.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://vbithyd.ac.in/"
+    ],
+    "name": "Vignana Bharathi Institute of science and Technology",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "vbithyd.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://www.ksriet.ac.in/"
+    ],
+    "name": "KSR Institute for engineering and technology",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "ksriet.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://www.lnmiit.ac.in/"
+    ],
+    "name": "The LNM Institute of Information Technology",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "lnmiit.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://ztu.edu.ua/"
+    ],
+    "name": "Zhytomyr Polytechnic",
+    "alpha_two_code": "UA",
+    "state-province": null,
+    "domains": [
+      "student.ztu.edu.ua"
+    ],
+    "country": "Ukraine"
+  },
+  {
+    "web_pages": [
+      "https://pstu.edu/en/"
+    ],
+    "name": "State Higher Educational Institution «Pryazovskyi State Technical University»",
```

Comment:
I guess the special chars `«` & `»` should be avoided for potential encoding/decoding issues.

`State Higher Educational Institution (Pryazovskyi State Technical University)`